### PR TITLE
add m_lama (multilingual lama) dataset

### PR DESCRIPTION
**Pull request by pdufter**
Add a multilingual (machine translated and automatically generated) version of the LAMA benchmark. For details see the paper https://arxiv.org/pdf/2102.00894.pdf

### CODE REVIEW

**Comment by pdufter** [comment]
Hi, it seems I am somewhat stuck here. The failed test `ci/circleci: run_dataset_script_tests_pyarrow_1_WIN` seems to be caused by some broken connection (`ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host`). Any help on this is appreciated.

**Comment by pdufter** [comment]
I guess the `dummy_data.zip` is too large. I can reduce the languages that are contained there, but when testing it, it obviously throws an error, as not all files can be found. I guess I can either i) change the default value regarding which languages are loaded or ii) let the `_generate_examples` silently skip any language for which it cannot find files. Both solutions are not really pretty - is there another way around this?

File: datasets/m_lama/README.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+@inproceedings{petroni2019language,
+  title={Language Models as Knowledge Bases?},
+  author={F. Petroni, T. Rockt{\"{a}}schel, A. H. Miller, P. Lewis, A. Bakhtin, Y. Wu and S. Riedel},
+  booktitle={In: Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing (EMNLP), 2019},
+  year={2019}
+}
+- **Leaderboard:**
+- **Point of Contact:**
+http://cistern.cis.lmu.de/mlama/
```
Comment (lhoestq) [inline datasets/m_lama/README.md:145]:
```suggestion
- **Homepage:** [Multilingual LAMA](http://cistern.cis.lmu.de/mlama/)
- **Repository:** [Github](https://github.com/norakassner/mlama)
- **Paper:** [Arxiv](https://arxiv.org/abs/2102.00894)
- **Point of Contact:** [Contact section](http://cistern.cis.lmu.de/mlama/)
```

We expect simple urls here

File: datasets/m_lama/README.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+This dataset contains data in 53 languages: 
+af,ar,az,be,bg,bn,ca,ceb,cs,cy,da,de,el,en,es,et,eu,fa,fi,fr,ga,gl,he,hi,hr,hu,hy,id,it,ja,ka,ko,la,lt,lv,ms,nl,pl,pt,ro,ru,sk,sl,sq,sr,sv,ta,th,tr,uk,ur,vi,zh
+
+## Dataset Structure
+For each of the 53 languages and each of the 43 relations/predicates there is a set of triples.
+
+### Data Instances
+
+Each triple consists of an object, a predicate and a subject. For each predicate there is a template available. 
+
```
Comment (lhoestq) [inline datasets/m_lama/README.md:177]: Could you provide one example from the dataset please here ?

To do so feel free to use `load_dataset` and take the first sample `dataset["train"][0]`

If it's too big you can crop it and only show the first fields, and add "..." at the end.

File: datasets/m_lama/README.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+Names of (most likely) famous people who have entries in Google Knowledge Graph or Wikidata.
+
+## Considerations for Using the Data
+
+Data was created through machine translation and automatic processes.
+
+### Social Impact of Dataset
+
+Unknown.
```
Comment (lhoestq) [inline datasets/m_lama/README.md:238]:
```suggestion
[More Information Needed]
```

File: datasets/m_lama/README.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+
+### Discussion of Biases
+
+Unknown.
```
Comment (lhoestq) [inline datasets/m_lama/README.md:242]:
```suggestion
[More Information Needed]
```

File: datasets/m_lama/README.md [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+  volume    = {abs/2102.00894},
+  year      = {2021},
+  url       = {https://arxiv.org/abs/2102.00894},
+  archivePrefix = {arXiv},
+  eprint    = {2102.00894},
+  timestamp = {Tue, 09 Feb 2021 13:35:56 +0100},
+  biburl    = {https://dblp.org/rec/journals/corr/abs-2102-00894.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org},
+  note      = {to appear in EACL2021}
+}
```
Comment (lhoestq) [inline datasets/m_lama/README.md:277]:
````suggestion
```
@article{kassner2021multilingual,
  author    = {Nora Kassner and
               Philipp Dufter and
               Hinrich Sch{\"{u}}tze},
  title     = {Multilingual {LAMA:} Investigating Knowledge in Multilingual Pretrained
               Language Models},
  journal   = {CoRR},
  volume    = {abs/2102.00894},
  year      = {2021},
  url       = {https://arxiv.org/abs/2102.00894},
  archivePrefix = {arXiv},
  eprint    = {2102.00894},
  timestamp = {Tue, 09 Feb 2021 13:35:56 +0100},
  biburl    = {https://dblp.org/rec/journals/corr/abs-2102-00894.bib},
  bibsource = {dblp computer science bibliography, https://dblp.org},
  note      = {to appear in EACL2021}
}
```
````
use code block

**Review by lhoestq — COMMENTED** [review]
Awesome ! Thanks for adding this one

This looks all good to me :)
I just added a few comments in the dataset card

Moreover about the size of the dummy_data, I can see that you already did a good job at making it small. Thank you for doing that !

I guess we can still reduce it a bit more by having only 1 line per text file instead of 5.
Could you do that if you don't mind ? If that's something that takes too much time we can keep it this way though...

File: datasets/m_lama/README.md (hunk 2 above)
Comment (pdufter) [reply datasets/m_lama/README.md:162]: Good idea, I added one example.

**Comment by pdufter** [comment]
Thanks for the review and the constructive comments :) ! I tried to address them, and reduced the number of lines in the dummy data to 1 to reduce its size.

**Review by lhoestq — APPROVED** [review]
Thanks for the changes :)

**Pull request merged**